cmd/network.go

```diff
@@ -534,6 +534,64 @@ func generateNetworkPolicies(profile *kubeflowv1.Profile) []*networkingv1.Networ
 		},
 	})
 
+	// Allow egress to Pipelines
+	port8888 := intstr.FromInt(8888)
+	port8887 := intstr.FromInt(8887)
+	policies = append(policies, &networkingv1.NetworkPolicy{
+		ObjectMeta: metav1.ObjectMeta{
+			Name:      "notebooks-pipelines-egress",
```

Comment:
```suggestion
			Name:      "notebooks-unclassified-pipelines-egress",
```

I would recommend we name this `notebook-unclassified-pipelines-egress` since it follows the naming conventions of the others.

Reply:
Done!